pgmoon/init.lua

```diff
@@ -6,8 +6,12 @@ do
   local _obj_0 = require("pgmoon.bit")
   rshift, lshift, band = _obj_0.rshift, _obj_0.lshift, _obj_0.band
 end
-local pl_file = require("pl.file")
-local ssl = require("ngx.ssl")
+local pl_file
+local ssl
+if ngx then
```

Comment:
`if ngx then ... end` is one way to check if are running in openresty

Reply:
The usual we we do this is to use `pcall(require, "foo")`, but I'm fine with `if ngx then` as well. No real difference here.